nf_core/pipeline-template/README.md

```diff
@@ -25,57 +25,76 @@
 
 ## Introduction
 
-<!-- TODO nf-core: Write a 1-2 sentence summary of what data the pipeline is for and what it does -->
+<!-- TODO nf-core: Write a 2-3 sentence summary of what data the pipeline is for and what it does, e.g.
 
-**{{ name }}** is a bioinformatics best-practice analysis pipeline for {{ description }}.
+   > **nf-core/rnaseq** is a bioinformatics pipeline that can be used to analyse RNA sequencing data obtained from
+   > organisms with a reference genome and annotation. It takes a samplesheet and FASTQ files as input,
+   > performs quality control (QC), trimming and (pseudo-)alignment, and produces a gene expression matrix and extensive QC report.
```

Comment:
```suggestion
   > **{{ name }}** is a bioinformatics pipeline that can.. Please complete these sentences with a description
   > of the types of data that the pipeline ingests, a brief overview of the major pipeline sections and the types of
   > outputs that it produces. You're giving an overview to someone new to nf-core here, in 15-20 seconds.
```

Reply:
Personally, I find an example much more instructive than the text (this is why stackoverflow took off in the first place). But I don't mind your version too much, or we could have both.

Reply:
I mostly just found it a bit jarring as I read through. Maybe we could put in a couple of links to other pipelines as examples?